platform/core.multitabs/src/org/netbeans/core/multitabs/impl/ButtonPopupSwitcher.java

```diff
@@ -549,4 +552,4 @@ private void selectTab(TabData tab) {
             }
         }
     }
-}
+}
```

Comment:
Unnecessary whitespace change.

Reply:
Not sure what white space, it is the end of file, I have not changed anything there.  Can you give me a hint please?

Reply:
looks like you removed the newline symbol after '}'. Its common to have one at the end. That is what the stop sign means on github.

You can fix this by putting the cursor after '}' and press return.

Reply:
Oh okay, I thougth it would be that, but looked into master and did not see newline at the end. Will add it there, thanks.